app/services/AuthService.ts

```diff
@@ -26,12 +26,12 @@ export const AuthService = {
 	 * @returns token or throws an exception
 	 */
 	async validateLogin(uid: string, password: string):Promise<String> {
-		return await axios.post('http://hslinux:38383/api/v1/auth', {
+		return await axios.post(process.env.LDAP_URL, {
```

Comment:
I'd like to see this process.env.LDAP_URL grabbed from the config folder. So we can have a default of sorts if we don't have it set as an environmental variable. Whatever default you set it to doesn't matter.